index.js

```diff
@@ -1,4 +1,4 @@
-import SafariDriver from './lib/driver';
+import * as SafariDriver from './lib/driver';
```

Comment:
this might break module exports. Please change to `import {SafariDriver}...`

Reply:
yea, i've noticed that.

Reply:
the difference in `import * as X` and `import {X}` is that the fist instruction imports all exported items from the given module and puts then into the X namespace and the second imports a single export with the same name.

For example, let say we have module M exporting class C:

```
export class C {}
export function f(){}
```

then 

```
// works
import {C} from 'M';
import {f} from 'M'
class CC extends C;
f();
```

```
// fails
import * as N from 'M';

// fails, because N is the namespace
class N extends C;
N();

// works
class N.C extends C;
N.f();
```